ui/components/GrafanaComponent.js

```diff
@@ -74,7 +74,7 @@ class GrafanaComponent extends Component {
 
       grafanaURL,
       grafanaAPIKey,
-      grafanaBoardSearch: '', // we probably dont need this retrieved from store
+      grafanaBoardSearch,
```

Comment:
removing the default value might cause bugs. Did you absolutely make sure it wasn't required?
I think it is being used in line 177 of the file.
Also, look at line 184 (https://github.com/layer5io/meshery/blob/a9ee4fd229c54c2de7dc0699dcd8501e8d0f2264/ui/components/GrafanaComponent.js#L184), we don't want to send undefined in the URL.

Reply:
Yes, it was not required, also it was null in the assignment. So I dont think sending empty value is better. Anyways it is assigned its proper value, you may test your local Grafana setup.

Reply:
Maybe you are right but as far as I know, non assigned variables have `undefined` as value. And using `undefined` as a variable in template string causes it to have undefined in string type.

Example:
```js
var aNonAssignedVariable;
var constructedString = `prefix${aNonAssignedVariable}suffix`;
console.log(constructedString) // "prefixundefinedsuffix"
```

Reply:
We absolutely don't want this to be sent in the query parameter. Empty string is way better and can be validated in the server side.

Reply:
Then there is no need to fetch BoardSearch prop in constructor from prop if we are ultimately giving it a empty string.

Reply:
That's exactly what the code comment on that statement says

Reply:
that's exactly the comment on the statement says.